entries/r/rippling.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Rippling": {
+    "domain": "rippling.com",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "finance"
```

Comment:
I'm not so sure about `finance` here. This software/service seems to cover a whole lot of categories, but nothing specifically either. Maybe `other` is suited better.

Reply:
@phallobst For what it's worth, I'd consider Rippling to be a competitor to both Gusto and ADP. Both of those entries use the `finance` keyword. I can add other, but if that's the case, you might consider updating the other entries as well.